backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/impexp/processing/DBFunctionHelper.java

```diff
@@ -45,6 +45,12 @@ public class DBFunctionHelper
 {
 	private static final transient Logger logger = LogManager.getLogger(DBFunctionHelper.class);
 
+	/**
+	 * Used to call import DB functions using c_dataimport_id and record_id as parameters
+	 * @param function
```

Comment:
pls avoid `@param` without a description.
If you don't have it, don't add the `@param`... adds no value/info.